src/components/lists/FamilyList.tsx

```diff
@@ -211,14 +216,22 @@ export default function FamilyList() {
             {filteredItems?.map((family) => (
               <Tr
                 key={family.import_id}
+                data-testid={`family-row-${family.import_id}`}
                 borderLeft={
                   family.import_id === familyError ? '2px' : 'inherit'
                 }
                 borderColor={
                   family.import_id === familyError ? 'red.500' : 'inherit'
                 }
               >
-                <Td>{family.title}</Td>
+                <Td>
+                  <Flex gap="2" alignItems="center">
+                    {!family.documents?.length && !family.events?.length && (
```

Comment:
Should this be `||` for the either of the lengths are 0? Or do we want to only show if both are 0?

Reply:
I understand that only if both are 0. @katybaulch please confirm this but I guess that the Family will be published if there is any event OR document

Reply:
or @alaniswright

Reply:
Patrick is correct - both are needed otherwise it will not be published. So it should be OR.

Design looks good! Will be hard for LSE to accidentally not publish things now :)

Reply:
@PatFawbertMills @alaniswright  ok, then please confirm this cases:
- If a Family has Events but hast NOT Documents -> NO PUBLISHED
- If a Family has NOT Events but has Documents -> NO PUBLISHED
- If a Family has NOT Events but NOT Documents -> NO PUBLISHED

Only it will be published if:
- If a Family has Events and has Documents -> PUBLISHED

Reply:
In the definition of the ticket it explained the following:

> "Some families added by LSE have not been published because they don't have an associated event or physical document."

https://linear.app/climate-policy-radar/issue/PDCT-831/make-it-easier-to-see-when-a-family-wont-get-published-because-of

Reply:
Yes correct: "if they don't have an event **or** a document" then we should not allow the document to be published

Reply:
Then it is correct (careful with double negations!)

Reply:
But you reaction to the cases I exposed is not the same you are saying here...

Reply:
Have I misunderstood? I read the code saying the length of both the documents and events needs to be 0 in order for the condition to pass (and show the warning)? Which seems like it won't pass if one or the other is > 0?

Reply:
Not 0 and not 0. (Check the exclamations)

Reply:
0 Documents and 0 Events (check the exclamations). This is ok for the cases defined in the previous comment

Reply:
I have added a new test below that is failing in order to illustrate the point